symplyphysics/laws/thermodynamics/work_done_by_gas_at_constant_pressure.py

```diff
@@ -0,0 +1,45 @@
+from sympy import Eq, solve
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+                           validate_output)
+
+# Description
+# Common formula for work done by gas is A = ∫p * dV, where
```

Comment:
It is better to make a separate law from this description and add high level explanation instead.